utils/src/main/java/org/corfudb/utils/lock/states/NoLeaseState.java

```diff
@@ -90,7 +91,13 @@ private void acquireLease() {
             if (lockStore.acquire(lock.getLockId())) {
                 lock.input(LockEvent.LEASE_ACQUIRED);
             }
-        } catch (Exception e) {
+        }
+        catch (WrongClusterException wce) {
```

Comment:
Can we have a test that verifies that LR is correctly restarted and able to resume?